### PR TITLE
Fix for 'Link for cloud-controller-manager is not working'

### PR DESCRIPTION
**Pull request by McGon-Fid**
Making a minor document link change in kubernetes autoscaler FAQ.md.

Issue is here: #5332

### CODE REVIEW

File: cluster-autoscaler/FAQ.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ More about Pod Priority and Preemption:
...
 
-It does _not_ delete the [Node object](https://kubernetes.io/docs/concepts/architecture/nodes/#api-object) from Kubernetes. Cleaning up Node objects corresponding to terminated instances is the responsibility of the [cloud node controller](https://kubernetes.io/docs/concepts/architecture/cloud-controller/#node-controller), which can run as part of [kube-controller-manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/) or [cloud-controller-manager](https://v1-19.docs.kubernetes.io/docs/reference/command-line-tools-reference/cloud-controller-manager/).
+It does _not_ delete the [Node object](https://kubernetes.io/docs/concepts/architecture/nodes/#api-object) from Kubernetes. Cleaning up Node objects corresponding to terminated instances is the responsibility of the [cloud node controller](https://kubernetes.io/docs/concepts/architecture/cloud-controller/#node-controller), which can run as part of [kube-controller-manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/) or [cloud-controller-manager](https://https://kubernetes.io/docs/concepts/architecture/cloud-controller/).
```
Comment (gjtempleton) [inline cluster-autoscaler/FAQ.md:237]: Duplication of `https://`:
```suggestion
It does _not_ delete the [Node object](https://kubernetes.io/docs/concepts/architecture/nodes/#api-object) from Kubernetes. Cleaning up Node objects corresponding to terminated instances is the responsibility of the [cloud node controller](https://kubernetes.io/docs/concepts/architecture/cloud-controller/#node-controller), which can run as part of [kube-controller-manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/) or [cloud-controller-manager](https://kubernetes.io/docs/concepts/architecture/cloud-controller/).
```

**Review by gjtempleton — CHANGES_REQUESTED** [review]
One slight fix required

**Comment by McGon-Fid** [comment]
@gjtempleton thank you. I've now committed the suggested change for review. 

Correcting URL syntax for document link change in kubernetes autoscaler FAQ.md.

Issue is here: https://github.com/kubernetes/autoscaler/issues/5332

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

**Pull request merged**